README.md

```diff
@@ -1,3 +1,25 @@
-# TODO
 
-Use this template when creating new typescript projects.
+# io-pay
+- [io-pay](#io-pay)
+  - [Prerequisites](#prerequisites)
+  - [Usage](#usage)
+  
+Is a [WISP](https://docs.italia.it/italia/pagopa/pagopa-specifichepagamenti-docs/it/stabile/_docs/SANP_2.2_Sez2_Cap06_ComponentiTecnicheNodo.html#componente-wisp) replacement project and it will be the first web interface to allow guest (_citizen_) payment of [pagoPa](https://www.pagopa.gov.it/) system.
+
+From now wont be true _"pagoPA non è un sito dove pagare"_ it'll be too 🚀
+
+## Prerequisites
+
+## Usage
+
+From one console typing :
+```sh
+yarn install
```

Comment:
```suggestion
$ yarn install
```

Reply:
why ? in this way you can copy/paste the command from `README`.
see https://github.com/pagopa/io-pagopa-node-mock for example

Reply:
We used to have the convention of prefixing the dollar sign for commands to be entered in a console, I guess the convention got lost over time, it's not super important